### PR TITLE
[CMS-1400] Page tree driven routing integration

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: core/views.py [hunk 1]
```diff
@@ -128,3 +135,44 @@ def get_context_data(self, *args, **kwargs):
             'heading'
         )
         return context
+
+
+class SetupGuideLandingPageCMSView(BaseCMSPage):
```
Comment (eadpearce) [inline core/views.py]: these views (SetupGuideLandingPageCMSView, SetupGuidePageCMSView, UKRegionPageCMSView) are holdovers from the old invest codebase that this repo was based on and can be removed

Comment (ababic) [reply core/views.py]: @eadpearce. In the spirit of keeping PRs small and focussed, would it be okay to create a card for cleanup tasks like this, and tackle them separately?

Comment (ababic) [reply core/views.py]: @eadpearce Please Ignore the above... I've just see that these views are being added by this PR when they shouldn't be. I'll get those removed.

**Comment by ababic** [comment]
@eadpearce Those stray views have been removed now (must've been something that creeped in during merge conflict resolution). Could I request a re-review please?

**Pull request merged**